fluentgen/src/main/java/com/azure/autorest/fluent/mapper/ExampleParser.java

```diff
@@ -162,25 +162,23 @@ private FluentExample getExample(Map<String, FluentExample> examples,
         FluentExample example = examples.get(name);
         if (example == null) {
             example = new FluentExample(CodeNamer.toPascalCase(groupName), CodeNamer.toPascalCase(methodName),
-                    this.aggregateExamples ? null : exampleName,
-                    clientMethod.getProxyMethod().getOperationId(),
-                    getApiVersion(clientMethod));
+                    this.aggregateExamples ? null : exampleName);
             examples.put(name, example);
         }
         return example;
     }
 
-    private static String getApiVersion(ClientMethod clientMethod) {
-        String apiVersion = clientMethod.getProxyMethod().getParameters().stream()
-                .filter(p -> "api-version".equals(p.getRequestParameterName()))
-                .map(ProxyMethodParameter::getDefaultValue)
-                .findFirst()
-                .orElse(null);
-        if (apiVersion == null) {
-            logger.warn("Failed to find api-version in method '{}'", clientMethod.getName());
-        }
-        return apiVersion;
-    }
+//    private static String getApiVersion(ClientMethod clientMethod) {
+//        String apiVersion = clientMethod.getProxyMethod().getParameters().stream()
+//                .filter(p -> "api-version".equals(p.getRequestParameterName()))
```

Comment:
why do we comment this? delete it ?

Reply:
I am not sure if I might need to use that again. So for now keep it as commented.

Reply:
OK deleted.